src/test/resources/messages.properties

```diff
@@ -9,6 +9,9 @@ invalid.past-or-present.date=${validatedValue} date couldn't be in the future
 invalid.password=Your password must be between 8 and 30 characters at must contain at least: 1 uppercase Latin character (A-Z), 1 lowercase Latin character (a-z), 1 digit (0-9)
 invalid.username.size=Your login must be between 2 and 50 characters, do not use space
 invalid.token=You must be authenticated
+token.not.start.with.bearer=JWT token is not starts with Bearer prefix
+token.signature.not.valid=JWT signature does not match locally computed signature. JWT validity cannot be asserted and should not be trusted.
+token.expired=JWT expired at
```

Comment:
If you want to verify it here you should also add this logic to the main code.

Reply:
This logic is contained in `validateToken(String token, HttpServletRequest req)` method. If token is expired `catch (JwtException e)` block is running and this message puts in `req.setAttribute("detailedError", e.getMessage());`